pam/pam.go

```diff
@@ -59,7 +61,8 @@ func sendReturnMessageToPam(mTx pam.ModuleTransaction, retStatus adapter.PamRetu
 		style = pam.TextInfo
 	}
 
-	_ = showPamMessage(mTx, style, msg)
+	err := showPamMessage(mTx, style, msg)
+	log.Warningf(context.TODO(), "Impossible to send PAM message: %v", err)
```

Comment:
This will always print, even if `showPamMessage` returns nil for `err`. You need to do some error check first.

Reply:
ouch, fixed!